### PR TITLE
Update PDS vs BGS distinction, add updated federation details, fix typos

### PR DESCRIPTION
**Pull request by emilyliu7321**
- Update the distinction between PDS vs BGS and small world vs big world; updates images 
- Add federation details and link to architecture overview blog post 
- Adds crosslink to Bluesky FAQ within atproto FAQ 
- Adds social links to Bluesky
- Standardizes short-hand references to the AT Protocol as `atproto`, not ATP 
- Fixes misc typos

### CODE REVIEW

File: content/specs/at-uri-scheme.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ summary: A URI scheme for addressing ATP repository data.
 }
 </style>
 
-# ATP URI Scheme
+# Atproto URI Scheme
```
Comment (bnewbold) [inline content/specs/at-uri-scheme.md:22]: I think for the URI schema specifically we should call it "AT URI Scheme". In the Lexicons, I think we call them "at-uri", and the prefix is `at://` not `atproto://`. Switching from `ATP` to `AT` in this title is helpful!

Also, just as some context, this page specifically has been linked to from an IETF request for provisional URI schema registration, so we should be particularly careful not to make large changes or move the URL of the page. The other `ATP -> atproto` changes on this page do look good to go.

File: content/guides/faq.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Other smaller differences include: a different viewpoint about how schemas shoul
 
 ## Why create Lexicon instead of using JSON-LD or RDF?
 
-ATP exchanges data and RPC commands across organizations. For the data and RPC to be useful, the software needs to correctly handle schemas created by separate teams. This is the purpose of [Lexicon](/guides/lexicon).
+Atproto exchanges data and RPC commands across organizations. For the data and RPC to be useful, the software needs to correctly handle schemas created by separate teams. This is the purpose of [Lexicon](/guides/lexicon).
 
 We want engineers to feel comfortable using and creating new schemas, and we want developers to enjoy the DX of the system. Lexicon helps us produce strongly typed APIs which are extremely familiar to developers and which provides a variety of runtime correctness checks (which are vital in distributed systems).
```
Comment (ianklatzco) [inline content/guides/faq.md:26]: expanding DX to "developer experience" might be clearer

Comment (emilyliu7321) [reply content/guides/faq.md:26]: thanks for the suggestion, just pushed a clarification for this!

File: content/specs/at-uri-scheme.md (hunk 1 above)
Comment (emilyliu7321) [reply content/specs/at-uri-scheme.md]: got it, helpful context thanks!

**Review by pfrazee — APPROVED** [review]
Did a quick skim, lgtm

**Review by pfrazee — APPROVED** [review]
Did a quick skim, lgtm

**Pull request merged**